index.html

```diff
@@ -1,20 +1,18 @@
-jessica Baker
-
-"About"
-My name is Jessica Baker. I am 30 years old. I live in Palatine Il
-with my 3 chrildren. I am a student at CTD school.My favorite subject
-is math. My hobbies are listening to music and taking my chrildren 
-out to fun places.
-
-"Experience"
--javascript
--linux
-
-"Connect"
-https://github.com/Jessica12492/intro-to-programming-dorado
-https://mail.google.com
-https://www.facebook.com
-
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Personal Portfolio Website</title>
+    <meta charset="UTF-8">
+    <section>Jessica Baker<h2>About</h2>My name is Jessica Baker. I am 30 years old. I live in Palatine il with my 3 children. I am a student at CTD school. My favorite subject is math. My hobbies  are listening to music , shopping and taking my children out to fun places.</section>
+     <section> <h2>Experience</h2> <ul><li>javascript</li><li>linux </li> </ul> </section>
+   <section><h3>Connect</h3><ul><li> https://github.com/Jessica12492/intro-to-programming-dorado</li> <li> https://mail.google.com</li> <li> https://www.facebook.com</li></ul>
+    </section>
+    <img src="../myself.jpg" alt="Picture of Me"
+    width="100"
+    height="90">
+</body>
```

Comment:
The part of the HTML document that contains the things that we want to show the user on the page, should be wrapped by a **body** tag. Right now you have the closing tag `</body>` but you're missing the opening tag `<body>`